examples/Reserve.php

```diff
@@ -14,6 +14,11 @@
 print_r($statistics);
 
 // Get the reserve ledger
-$ledger = $client->getReserve()->getLedger();
+$ledgerPager = $client->getReserve()->getLedger();
```

Comment:
@fixe In the same file, on line 22 we will have another `$pager` and that doesn't seems right.